.github/workflows/build.yaml

```diff
@@ -123,18 +123,11 @@ jobs:
         run: |
           mkdir ${{ env.RELEASE_PATH }}
           cp ./dist/${{ matrix.target_os}}_${{ matrix.target_arch}}/release/rad.exe ${{ env.RELEASE_PATH }}/rad_${{ matrix.target_os}}_${{ matrix.target_arch}}.exe
-      - name: Upload Release binaries 
```

Comment:
Is this being done elsewhere now?

Reply:
We uploaded same binary twice with different name. This change will upload it once.